src/avatar/src/Avatar.js

```diff
@@ -62,6 +64,7 @@ const Avatar = memo(
     )
 
     const [imageHasFailedLoading, setImageHasFailedLoading] = useState(false)
+    const onError = useCallback(() => setImageHasFailedLoading(true), [])
```

Comment:
this may not be as efficient as one would hope: https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
Yeah for sure, might be too aggressive here. I was considering that there is internal state that will cause rerenders, and so we may as well cache the function, but it might be excessive in this case.